main.js

```diff
@@ -947,20 +1038,53 @@ function start() {
         saveAs(blob, "system_information_963", true);
         flowey_laugh_once();
     }, false);
+    
+    var allowedLocations1 = document.getElementById("allowed-locations");
+    var allowedLocations2 = document.getElementById("allowed-locations-2");
+    allowedLocations1.addEventListener("change", function() {
+        allowedLocations2.value = allowedLocations1.value;
+        updateSelection("ini-location", document.getElementById("ini-location").value, rooms[allowedLocations1.value]);
+        updateSelection("sav-location", document.getElementById("sav-location").value, rooms[allowedLocations1.value]);
+    }, false);
+    allowedLocations2.addEventListener("change", function() {
+        allowedLocations1.value = allowedLocations2.value;
+        updateSelection("ini-location", document.getElementById("ini-location").value, rooms[allowedLocations1.value]);
+        updateSelection("sav-location", document.getElementById("sav-location").value, rooms[allowedLocations1.value]);
+    }, false);
+    
     var weaponSelect = document.getElementById("sav-weapon");
+    var armorSelect = document.getElementById("sav-armor");
     weaponSelect.onchange = function() {
```

Comment:
Could this be `weaponSelect.addEventListener("change", function() {...});` like in the rest of the code?